src/main.cc

```diff
@@ -211,6 +212,8 @@ int main(int argc, char *argv[])
     QGCApplication* app = new QGCApplication(argc, argv, runUnitTests);
     Q_CHECK_PTR(app);
 
+    QApplication::setWindowIcon(QIcon(":/res/resources/icons/qgroundcontrol.ico"));
```

Comment:
Famous last words :) I'd rather not take the chance. Otherwise I'll need to test all OS.